recipes/h3/all/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 cmake_minimum_required(VERSION 3.1)
 project(cmake_wrapper)
 
+set(CMAKE_C_STANDARD 99)
```

Comment:
You should use targets, global is not recommended.

Reply:
It's easier to maintain like that, since I've added an option to build executables (filters), and we can't predict if some executables will be added or removed in new versions.
Moreover H3 globally enables C99 in their toolchain file: https://github.com/uber/h3/blob/v3.6.4/cmake/toolchain.cmake

The book `Professional CMake, A Practical Guide`  (§15.1) claims that global LANG STANDARD is not a bad practice.